onward/app/controllers/CardController.scala

```diff
@@ -37,6 +37,7 @@ class CardController(
       "www.dailymotion.com",
       "youtube.com",
       "www.youtube.com",
+      "www.youtube-nocookie.com",
```

Comment:
I don't know if that's necessary, but thought it wouldn't hurt anyway. Can remove if not needed.